pywbem/_mof_compiler.py

```diff
@@ -2834,7 +2861,11 @@ def compile_string(self, mof, ns, filename=None):
           ns (:term:`string`):
             The name of the CIM namespace in the associated CIM repository
             that is used for lookup of any dependent CIM elements, and that
-            is also the target of the compilation.
+            is also the target of the compilation in the MOF file/string being
+            compiled. If `None`, the default namespace of the connection is
+            used. A namespace defined in a namespace pragma of the MOF
+            has precedence over the namespace parameter.  The namespace
+            specified in this parameter or the MOF itself must exist.
```

Comment:
he notion of "precedence" does not sufficiently explain it. See my previous comment.